src/main/java/com/abrus/user/model/User.java

```diff
@@ -0,0 +1,31 @@
+package com.abrus.user.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "user")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String email;
+    private String phone;
+
+    @ManyToMany(mappedBy = "assignees")
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private List<Role> roles;
```

Comment:
Optional: for `@ManyToMany` it is better to use Set, not List
```suggestion
    private Set<Role> roles;
```